src/components/MessageListener/MessageListener.js

```diff
@@ -0,0 +1,28 @@
+import { useEffect } from 'react';
+import { useSelector } from 'react-redux';
+
+const useListenMessage = (callback, eventType = 'trafimage-event') => {
+  const tm = document.getElementsByTagName('trafimage-maps')[0];
```

Comment:
I dont' like so much this hardoded stuff. Could the MesageListener be  used in the WebComponent.js file ? and using refs you pass the html element ?

Reply:
or in TopicElements you already have a node there